ce/ce/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Shouldn't this be at `LICENSE` instead of `ce/ce/LICENSE`?

Reply:
This is a Node/npm artifact; ce is getting emitted as an npm package here so it needs its own copy of the license. (And I didn't think deduping this file was worth digging through npm's guts to tell it to find it somewhere else)

Reply:
It can get dropped once it's building the static binary. Since there isn't any standalone usage anymore